src/components/test/containers-test.js

```diff
@@ -70,3 +70,30 @@ describe('Actions', () => {
     assert.isTrue(wrapper.find('div.Hyp-Actions--column').exists());
   });
 });
+
+describe('Scrollbox', () => {
+  const createComponent = (props = {}) =>
+    mount(
+      <Scrollbox {...props}>
+        <div>This is content inside of a Scrollbox</div>
+      </Scrollbox>
+    );
+
+  it('renders children inside of a div with appropriate classnames', () => {
+    const wrapper = createComponent();
+
+    assert.isTrue(wrapper.find('div').first().hasClass('Hyp-Scrollbox'));
+  });
+
+  it('applies extra classes', () => {
+    const wrapper = createComponent({ classes: 'foo bar' });
+
+    assert.isTrue(wrapper.find('div.Hyp-Scrollbox.foo.bar').exists());
```

Comment:
If you want to check the correct class order, you can use this complicated test:
```suggestion
    assert.deepEqual(
      [
        ...wrapper
          .find('div.Hyp-Scrollbox.foo.bar')
          .getDOMNode()
          .classList.values(),
      ],
      ['Hyp-Scrollbox', 'foo', 'bar' ]
    );
```

Reply:
I've added a note to https://github.com/hypothesis/frontend-shared/issues/129 that could help to address this: I'd like to craft some common/reusable tests for components that take these common props.